config/sequelize.js

```diff
@@ -2,6 +2,7 @@ exports.default = {
     sequelize: function(api){
         return {
             "autoMigrate" : true,
+            "loadFixtures": true,
```

Comment:
Behaviour in previous versions performed the `loadFixtures` functionality automatically - this would mean we're changing the defaults?

If we're going down this direction, I'll suggest changing the default behaviour when `loadFixtures` config is not defined to `false`. This might break existing users upon upgrading, though.